src/components/scenes/DefaultFiatSettingScene.tsx

```diff
@@ -76,8 +76,12 @@ export class DefaultFiatSettingComponent extends React.Component<Props, State> {
 
   render() {
     const filteredArray = this.props.supportedFiats.filter(entry => {
+      const key = `currency_label_${entry.value}`
+      const subTitle = lstrings[key as keyof typeof lstrings] ?? ''
+
```

Comment:
We usually build a table for this type of thing, so like:

```ts
const fiatTitles = {
  USD: lstrings.currency_label_usd,
  EUR: lstrings.currency_label_eur,
  ...
}
```

That was we can't "escape" into the broader strings table, because we do the lookup on a safe subset.

Now, while I would prefer to see it done this way, I won't insist on it because we are really close to the merge and we want to get this in.